File: practice-2017-10-04-android/MyApplication/domain/src/main/java/com/example/jboy/entities/Cat.java
package com.example.jboy.entities;

import java.util.UUID;

/**
 * Created by Dev on 04/10/2017.
 */

public class Cat {
    private String name;
    private String title;
    private UUID id;

    public Cat(UUID id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public UUID getId() {
        return id;
    }
}

File: 2018-01-PSITS-SultanKudarat/practice-2018-01-13/TwoLayers/Cat.cs
using System;

namespace TwoLayers
{
    public class Cat
    {
        public Cat(Guid id)
        {
            Id = id;
        }

        public Guid Id { get; private set; }
        public string Name { get; set; }
        public string Title { get; set; }
    }
}